neqo-qpack/src/decoder.rs

```diff
@@ -696,7 +710,7 @@ mod tests {
             (String::from("my-headera"), String::from("my-valuea")),
             (String::from("my-headerb"), String::from("my-valueb")),
         ];
-        let header_block = &[0x03, 0x81, 0x10, 0x11];
+        let header_block_1 = &[0x03, 0x81, 0x10, 0x11];
```

Comment:
Do these need to be variables rather than

```suggestion
        const HEADER_BLOCK_1: &[u8] = &[0x03, 0x81, 0x10, 0x11];
```

Note that clippy will then complain about this not being at the top of the function.